components/admin_console/feature_discovery/feature_discovery.tsx

```diff
@@ -108,41 +158,7 @@ export default class FeatureDiscovery extends React.PureComponent<Props, State>
                             defaultMessage={copyDefault}
                         />
                     </div>
-                    <button
-                        type='button'
-                        className='btn btn-primary'
-                        data-testid='featureDiscovery_primaryCallToAction'
-                        onClick={this.requestLicense}
-                    >
-                        <LoadingWrapper
-                            loading={this.state.gettingTrial}
-                            text={Utils.localizeMessage('admin.license.trial-request.loading', 'Getting trial')}
-                        >
-                            <FormattedMessage
-                                id='admin.ldap_feature_discovery.call_to_action.primary'
-                                defaultMessage='Start trial'
-                            />
-                        </LoadingWrapper>
-                    </button>
-                    <a
-                        className='btn btn-secondary'
-                        href={learnMoreURL}
-                        data-testid='featureDiscovery_secondaryCallToAction'
-                        target='_blank'
-                        rel='noopener noreferrer'
-                    >
-                        <FormattedMessage
-                            id='admin.ldap_feature_discovery.call_to_action.secondary'
-                            defaultMessage='Learn more'
-                        />
-                    </a>
-                    {gettingTrialError}
-                    <p className='trial-legal-terms'>
-                        <FormattedMarkdownMessage
-                            id='admin.license.trial-request.accept-terms'
-                            defaultMessage='By clicking **Start trial**, I agree to the [Mattermost Software Evaluation Agreement](!https://mattermost.com/software-evaluation-agreement/), [Privacy Policy](!https://mattermost.com/privacy-policy/), and receiving product emails.'
-                        />
-                    </p>
+                    {this.props.prevTrialLicense && this.props.prevTrialLicense.IsLicensed === 'true' ? Utils.renderPurchaseLicense() : this.renderStartTrial(learnMoreURL, gettingTrialError)}
```

Comment:
I love me some TypeScript optional chaining 😸 !!
```suggestion
                    {this.props.prevTrialLicense?.IsLicensed === 'true' ? Utils.renderPurchaseLicense() : this.renderStartTrial(learnMoreURL, gettingTrialError)}
```

Reply:
Done.